Cargo.toml

```diff
@@ -20,15 +20,15 @@ exclude = ["soroban-test-wasms/wasm-workspace"]
 # NB: When bumping the major version make sure to clean up the
 # code guarded by `unstable-*` features to make it enabled
 # unconditionally.
-version = "20.2.2"
+version = "20.3.0"
 rust-version = "1.74.0"
 
 [workspace.dependencies]
```

Comment:
Does the bump version workflow need to be modified to include `soroban-simulation` crate?

Reply:
IIUC the version is set for all crates in the workspace. There are also workspace dependency bumps below, but `soroban-simulation` is not a dependency of any other crate in the workspace.